myhpom/templates/myhpom/content/why-plan.html

```diff
@@ -0,0 +1,237 @@
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-6 col-md-3">
+                    <div class="why-plan-overview__link-container">
+                        <div class="why-plan-overview__link">
+                            <a href="#ownership">Ownership</a>
+                        </div>
+                        <div class="why-plan-overview__link-text">
+                            It empowers you to make your own healthcare decisions.
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-md-3">
+                    <div class="why-plan-overview__link-container">
+                        <div class="why-plan-overview__link">
+                            <a href="#communication">Communication</a>
+                        </div>
+                        <div class="why-plan-overview__link-text">
+                            It allows you a clear, concise way to share your choices.
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-md-3">
+                    <div class="why-plan-overview__link-container">
+                        <div class="why-plan-overview__link">
+                            <a href="#responsibility">Responsibility</a>
+                        </div>
+                        <div class="why-plan-overview__link-text">
+                            Reduces difficulty on yourself and those you care about.
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-md-3">
+                    <div class="why-plan-overview__link-container">
+                        <div class="why-plan-overview__link">
+                            <a href="#peace-of-mind">Peace of mind</a>
+                        </div>
+                        <div class="why-plan-overview__link-text">
+                            Brings confidence that your wishes will be heard.
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="why-plan-bigimg" id="ownership">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6">
+                    <img src="{% static 'myhpom/img/why-plan/mmh-whyplan-markinset1.png' %}" alt="" class="why-plan-bigimg__img">
+                </div>
+                <div class="col-md-6">
+                    <h2 class="why-plan-bigimg__title">
+                        It Empowers You to Own Your Health
+                    </h2>
+                    <h3 class="h4 why-plan-bigimg__subtitle">
+                        Make Your Own Healthcare Decisions
+                    </h3>
+                    <div class="mt-3">
+                        {% scribble 'why-plan' 'empowerment' %}
+                        {% lorem 80 w %}
```

Comment:
(copy of slack comment about content) Did we decide that Shane should input the initial content? i.e., Since we have a draft of the content for this page, should we just go ahead and put it in?

https://drive.google.com/drive/u/0/folders/1iXVpwqp-0o0dzYMiPtHNsRwmHZf42nbA